### PR TITLE
Issue #1668 - Lowered log level and made messages less aggressive.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
Also fixed module info for pgsql and oracle modules.

Signed-off-by: Tomas Kraus <tomas.kraus@oracle.com>

### CODE REVIEW

File: foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/databaseaccess/JsonPlatformManager.java [hunk 1]
```diff
@@ -15,42 +15,88 @@
...
+            final Map<Class<? extends DatabasePlatform>, Supplier<DatabaseJsonPlatform>> fallbackMap, final String className) {
+        final Class<JsonPlatformProvider> providerClass = getProvider(className);
+        if (providerClass != null) {
+            try {
+                final JsonPlatformProvider provider  = providerClass.getConstructor().newInstance();
+                final Map<Class<? extends DatabasePlatform>, Supplier<DatabaseJsonPlatform>> platformsMap = provider.platforms();
+                if (platformsMap != null) {
+                    fallbackMap.putAll(platformsMap);
+                }
+            } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException | InstantiationException ex) {
```
Comment (lukasj) [inline foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/databaseaccess/JsonPlatformManager.java:63]: what about `ReflectiveOperationException`?

File: foundation/org.eclipse.persistence.core/src/main/java/module-info.java [hunk 2]
```diff
@@ -38,6 +38,8 @@
     requires static jakarta.messaging;
     requires static jakarta.resource;
 
+    uses org.eclipse.persistence.internal.databaseaccess.spi.JsonPlatformProvider;
```
Comment (lukasj) [inline foundation/org.eclipse.persistence.core/src/main/java/module-info.java:41]: so the core module will find it; what about eclipselink.jar?

Comment (Tomas-Kraus) [reply foundation/org.eclipse.persistence.core/src/main/java/module-info.java:41]: Let me guess ...it won't find it.

File: bundles/eclipselink/src/main/java/module-info.java [hunk 3]
```diff
@@ -45,6 +45,8 @@
 
     requires static com.sun.xml.bind.core;
 
+    uses org.eclipse.persistence.internal.databaseaccess.spi.JsonPlatformProvider;
```
Comment (lukasj) [inline bundles/eclipselink/src/main/java/module-info.java:48]: the flow of instructions in other descriptors is `requires`, `exports` and then `uses`. What is the reason for breaking the consistency here?

Comment (lukasj) [reply bundles/eclipselink/src/main/java/module-info.java:48]: see ie https://dev.java/learn/introduction-to-modules-in-java/

**Pull request merged**